selinux: fix exception, remove hardcoded strings

lasers opened the pull request:
`except OSError:` -> `except:` makes the status work. No more exception... Disabled tho.
It's technically still true because of `show when no SELinux policy is loaded.`

Few strings in there too for full-fledged customization.

Not sure if I should make a minor exception issue on this one.
Untested.

tobes commented:
> except OSError: -> except: makes the status work. No more exception...

what was the actual exception?

It is nice when we have well defined try/except blocks, poorly defined ones lead to really hard to find bugs.  Here it's not such an issue as there is no code.

> Few strings in there too for full-fledged customization.

yeah this I'm still not set on how about.

lasers commented:
```
2017-02-01 04:51:07 WARNING Instance `selinux`, user method `selinux_status` failed (AttributeError) selinux.py line 42.
2017-02-01 04:51:07 INFO Traceback
AttributeError: module 'selinux' has no attribute 'security_getenforce'
  File "/home/chris/src/py3status/py3status/module.py", line 584, in run
    response = method()
  File "/home/chris/src/py3status/py3status/modules/selinux.py", line 42, in selinux_status
    if selinux.security_getenforce():
```

tobes commented:
That exception is odd - is this about the version of selinux used?  I've not looked at the code

> Good for icons, language translations, phrasing differently, and such?

It's all about the name

tobes commented:
To me if your strings  in this particular module were `state_..` that would make sense and be in line with say the mpris module and also it fits because it is used in the `{state}` placeholder.

Anyhow I still think this needs a proper discussion rather than filling the Pull queue with `string_...` PRs that I will block as much as I can.  This is also a shame when the PR contains other code that I think should be merged.

lasers commented:
I added checking for `not-installed` here to help differentiate between `selinux: isn't installed` and `selinux: disabled`. 

This does not work for me. Confused. I moved it out of `post_config_hook()` to under `selinux()`... and it worked. There may be some kind of bug... maybe race condition somewhere.

tobes commented on py3status/modules/selinux.py:
```diff
@@ -2,32 +2,37 @@
+    string_enforcing: show when SELinux security policy is enforced.
+        (default 'enforcing')
+    string_permissive: show when SELinux prints warnings instead of enforcing.
+        (default 'permissive')
```
`state_...`

tobes commented on py3status/modules/selinux.py:
```diff
@@ -36,26 +41,31 @@ class Py3status:
+        if not self.py3.check_commands('getenforce'):
+            return {'cache_until': self.py3.CACHE_FOREVER,
+                    'color': self.py3.COLOR_BAD,
+                    'full_text': string_unavailable}
```
As elsewhere this is not how `post_config_hook()` works.  Sometimes it is good to just do one PR first so you get to know how it all works and we can fix things up as needed.  Currently you are creating work for both of us ;)

lasers replied:
@tobes IRC first.
